ibis/client.py

```diff
@@ -982,6 +1000,180 @@ def _get_schema_using_query(self, query):
 
         return ir.Schema(names, ibis_types)
 
+    def create_udf(self, udf_info, udf_name=None, db=None):
+        """
+        Creates a function within Impala
+
+        Parameters
+        ----------
+        udf_info : UDFCreator object
+        udf_name : string (optional)
+        db : string (optional)
+        """
+        name = udf_info.get_name()
+        if udf_name:
+            name = udf_name
+        statement = ddl.CreateFunction(udf_info.hdfs_file,
+                                       udf_info.so_symbol,
+                                       udf_info.inputs,
+                                       udf_info.output,
+                                       name, db)
+        self._execute(statement)
+
+    def create_uda(self, uda_info, uda_name=None, db=None):
+        """
+        Creates a user-defined aggregate function within Impala
+
+        Parameters
+        ----------
+        uda_info : UDAInfo object
+        uda_name : string (optional)
+        db : string (optional)
+        """
+        name = uda_info.get_name()
+        if uda_name:
+            name = uda_name
+        statement = ddl.CreateAggregateFunction(uda_info.hdfs_file,
+                                                uda_info.input_type,
+                                                uda_info.output_type,
+                                                uda_info.init_fn,
+                                                uda_info.update_fn,
+                                                uda_info.merge_fn,
+                                                uda_info.finalize_fn,
+                                                name,
+                                                db=db)
+        self._execute(statement)
+
+    def drop_udf(self, name=None, input_types=None, db=None, force=False,
```

Comment:
Use `database` instead of `db` to be consistent to other functions in this module